source/renderer/app/themes/daedalus/cardano.js

```diff
@@ -166,6 +166,7 @@ export const CARDANO_THEME_OUTPUT = {
     '--theme-connecting-background-color7': 'rgba(32, 34, 37, 0.91)',
     '--theme-connecting-background-color8': 'rgba(32, 34, 37, 1)',
     '--theme-connecting-text-color': '#fff',
+    '--theme-connecting-description-color': 'rgba(250, 251, 252, 0.7)',
```

Comment:
'--theme-connecting-description-color': 'rgba(255, 255, 255, 0.7)',

Reply:
@alexander-rukin could you please update Zeplin as well?
![Screen Shot 2021-12-08 at 08 34 10](https://user-images.githubusercontent.com/2846918/145201955-a1cc5446-d575-4820-b1c0-29b05f576ab1.png)

Reply:
why? it's not a cardano-theme. It's light blue

Reply:
You're right!